content/challenges/easy/7 - Singular or Plural/func.js

```diff
@@ -9,5 +9,5 @@
  * @returns {boolean}
  */
 module.exports = (word) => {
-    
+    return word.toLowerCase().endsWith('s')
```

Comment:
two other ways I"d like to see you solve this.
* treating word as a char[] (array of characters)
* using substring method